doc/tutorial/extending_theano.txt

```diff
@@ -259,7 +259,7 @@ Op Example
 
         def make_node(self, x):
             # check that the theano version has support for __props__
-            assert hasattr(self, '_props')
+            assert hasattr(self, '__props__')
```

Comment:
@carriepl what do you think? I think we should keep the old one. The goal is to know if the Theano version is recent enough. Using self.__props__ won't do that, as the Op will have it implemented.